pkg/namesgenerator/names-generator.go

```diff
@@ -257,6 +258,9 @@ var (
 		// Bailey Whitfield Diffie - American cryptographer and one of the pioneers of public-key cryptography. https://en.wikipedia.org/wiki/Whitfield_Diffie
 		"diffie",
 
+		// A. K. (Alexander Keewatin) Dewdney, Canadian mathematician, computer scientist, author and filmmaker. Contributor to Scientific American's "Computer Recreations" from 1984 to 1991. Author of Core War (program), The Planiverse, The Armchair Universe, The Magic Machine, The New Turing Omnibus, and more. https://en.wikipedia.org/wiki/Alexander_Dewdney
+		"dewdney",
```

Comment:
Could you replace one of these new names with a female? See this comment at the top of the file; https://github.com/moby/moby/blob/9d61eee75a3681fdff7f0c7e3232f4ccd3dfcaf6/pkg/namesgenerator/names-generator.go#L106

Reply:
Sure! I'll prep a new patch. Actually I can think of a few to add.